.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+nwjs_build_type=sdk
```

Comment:
Putting this here means that the `nw` devDep does not need to have `-sdk`. Better for dependency updating tools like dependabot